Composer/packages/client/src/components/Auth/AuthCard.tsx

```diff
@@ -0,0 +1,168 @@
+    fontWeight: 600,
+  },
+  email: {
+    display: 'block',
+    fontSize: FontSizes.small,
+  },
+  link: {
+    root: {
+      display: 'block',
+      fontSize: FontSizes.small,
+      marginTop: 5,
+    },
+  },
+  logoutLink: {
+    root: {
+      display: 'block',
+      fontSize: FontSizes.small,
+    },
+  },
+};
+
+export const AuthCard: React.FC = () => {
+  const [authCardVisible, setAuthCardVisible] = useState<boolean>(false);
+  const { refreshLoginStatus, requireUserLogin, logoutUser, setShowAuthDialog, setShowTenantDialog } = useRecoilValue(
+    dispatcherState
+  );
+  const isAuthenticated = useRecoilValue(isAuthenticatedState);
+  const currentUser = useRecoilValue(currentUserState);
+  const showAuthDialog = useRecoilValue(showAuthDialogState);
+  const showTenantDialog = useRecoilValue(showTenantDialogState);
+  const requiresGraph = useRecoilValue(requiresGraphState);
+
+  const personaRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    refreshLoginStatus();
+  }, []);
+
+  const logout = () => {
+    logoutUser();
+  };
+
+  const toggleAuthCardVisibility = () => {
+    setAuthCardVisible(!authCardVisible);
+  };
+
+  const switchTenants = () => {
+    setTenantId('');
+    requireUserLogin();
+    toggleAuthCardVisibility();
+  };
+
+  return (
+    <Fragment>
+      {/* this is the icon that appears at the top of the header bar */}
+      <div ref={personaRef}>
+        <Persona
+          hidePersonaDetails
+          componentRef={personaRef}
+          initialsColor={currentUser ? NeutralColors.gray160 : undefined}
+          size={PersonaSize.size32}
+          text={currentUser?.name}
+          onClick={toggleAuthCardVisibility}
+        />
+      </div>
+
+      {/* this is the actual login card */}
+      {authCardVisible && (
+        <Callout
+          gapSpace={10}
+          isBeakVisible={false}
+          styles={{
+            root: { width: 320, zIndex: zIndices.authContainer, backgroundColor: NeutralColors.white },
+          }}
+          target={personaRef.current}
+          onDismiss={toggleAuthCardVisibility}
+        >
+          {isAuthenticated ? (
+            <Stack tokens={{ childrenGap: 10, padding: 20 }}>
+              <Stack.Item align="end">
+                <Link styles={styles.logoutLink as ILinkStyles} onClick={logout}>
+                  {formatMessage('Sign out')}
+                </Link>
+              </Stack.Item>
+              <Stack.Item>
+                <Stack horizontal tokens={{ childrenGap: 20 }}>
+                  <Stack.Item>
+                    <Persona hidePersonaDetails initialsColor={NeutralColors.gray160} text={currentUser?.name} />
+                  </Stack.Item>
+                  <Stack.Item>
+                    <div style={styles.username}>{currentUser.name}</div>
+                    <div style={styles.email}>{currentUser.email}</div>
+                    <Link
+                      href={'https://myaccount.microsoft.com/'}
```

Comment:
does this need to be a aka.ms link?

Reply:
> @benbrown I was playing with the build of this branch. Functionality looks great!
> 
> One thing I did see was the cursor on the AuthCard is a crosshair instead of a pointer. In addition, is there no selected state on the button ?
> ![Screen Shot 2021-07-07 at 9 25 30 AM](https://user-images.githubusercontent.com/13004779/124795798-4c211880-df05-11eb-831b-45ec8f432930.png)

Fixed the cursor here.  There is not an active state as far as I know. @emivers8 care to chime in?

Reply:
No active state needed.